client/ayon_core/hosts/max/plugins/publish/validate_mesh_has_uv.py

```diff
@@ -0,0 +1,57 @@
+
+import pyblish.api
+from ayon_core.hosts.max.api.action import SelectInvalidAction
+from ayon_core.pipeline.publish import (
+    ValidateMeshOrder,
+    OptionalPyblishPluginMixin,
+    PublishValidationError
+)
+from pymxs import runtime as rt
+
+
+class ValidateMeshHasUVs(pyblish.api.InstancePlugin,
+                         OptionalPyblishPluginMixin):
+
+    """Validate the current mesh has UVs.
+
+    This validator only checks if the mesh has UVs but not
+    whether all the individual faces of the mesh have UVs.
+
+    It validates whether the current mesh has texture vertices.
+    If the mesh does not have texture vertices, it does not
+    have UVs in Max.
+
+    """
+
+    order = ValidateMeshOrder
+    hosts = ['max']
+    families = ['model']
+    label = 'Validate Mesh Has UVs'
+    actions = [SelectInvalidAction]
+    optional = True
+
+
+    @classmethod
+    def get_invalid(cls, instance):
+        invalid = [member for member in instance.data["members"]
+                   if not member.mesh.numTVerts > 0]
```

Comment:
What happens here if there's a member in the instance that is not a mesh? does it error on the `member.mesh` access?

Reply:
if there is instance which is not mesh, yes it will error out.

Reply:
Do we want that? It will likely be quite an unclear error to the user as well?

Reply:
> Do we want that? It will likely be quite an unclear error to the user as well?

we need to also build a check on this, i will make an update later